homeassistant/components/rfxtrx/config_flow.py

```diff
@@ -1,12 +1,251 @@
 """Config flow for RFXCOM RFXtrx integration."""
 import logging
 
+import voluptuous as vol
+
 from homeassistant import config_entries
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_COMMAND_OFF,
+    CONF_COMMAND_ON,
+    CONF_DEVICE,
+    CONF_DEVICE_ID,
+    CONF_DEVICES,
+)
+from homeassistant.core import callback
+from homeassistant.helpers.device_registry import (
+    async_entries_for_config_entry,
+    async_get_registry,
+)
 
-from . import DOMAIN
+from . import DOMAIN, get_device_id, get_rfx_object
+from .binary_sensor import supported as binary_supported
+from .const import (
+    CONF_AUTOMATIC_ADD,
+    CONF_DATA_BITS,
+    CONF_DEBUG,
+    CONF_FIRE_EVENT,
+    CONF_OFF_DELAY,
+    CONF_SIGNAL_REPETITIONS,
+    DEVICE_PACKET_TYPE_LIGHTING4,
+)
+from .cover import supported as cover_supported
+from .light import supported as light_supported
+from .switch import supported as switch_supported
 
 _LOGGER = logging.getLogger(__name__)
 
+CONF_OFF_DELAY_ENABLED = CONF_OFF_DELAY + "_enabled"
+
+
+class OptionsFlow(config_entries.OptionsFlow):
+    """Handle Rfxtrx options."""
+
+    def __init__(self, config_entry: ConfigEntry) -> None:
+        """Initialize vizio options flow."""
```

Comment:
Stale comment

Reply:
Good old copy/paste error...I actually pasted it from your rfxtrx_option_flow branch 😆 

I'll remove it :)